libs/vue-plugin/src/schematics/application/files/src/shims-tsx.d.ts.template

```diff
@@ -0,0 +1,13 @@
+import Vue, { VNode } from "vue";
```

Comment:
Does JSX work (I assume you need Babel configured)? If it doesn't, should we remove all references to `.tsx/.jsx`?

Reply:
It doesn't. Babel is required. It's also required for [modern mode](https://cli.vuejs.org/guide/browser-compatibility.html#modern-mode), which is currently listed as a builder option. Modern mode is a pretty fantastic feature, so maybe Babel gets added.